src/zope/testrunner/formatter.py

```diff
@@ -788,12 +790,10 @@ def id(self):
 # separately for richer error messages.
 try:
     import testtools
-    from testtools.content import (
-        Content,
-        ContentType,
-        content_from_file,
-        text_content,
-    )
+    from testtools.content import Content
+    from testtools.content import ContentType
+    from testtools.content import content_from_file
+    from testtools.content import text_content
```

Comment:
Oh, does our isort configuration mandate one import per line?

Why are we doing that to ourselves?

Reply:
I don't know. I am just blindly using `tox -eisort-apply`.
The rules are defined here: https://github.com/zopefoundation/meta/blob/d8ee00613a60c98a8d2227b0564403d0e447741a/config/default/setup.cfg.j2#L49-L61 – I personally use different rules (no groups of imports, just sort all imports alphabetically) but I do not care as long there is a tool to order the imports as defined by the rules.